packages/editor/src/components/entities-saved-states/entity-type-list.js

```diff
@@ -56,7 +56,11 @@ function GlobalStylesDescription( { record } ) {
 	);
 	return globalStylesChanges.length ? (
 		<PanelRow className="entities-saved-states__change-summary">
-			{ globalStylesChanges.join( ', ' ) }.
+			<ul className="entities-saved-states__changes">
+				{ globalStylesChanges.map( ( change ) => (
+					<li key={ change }>{ change }</li>
+				) ) }
+			</ul>
```

Comment:
```suggestion
			{ /*
			 * Disable reason: The `list` ARIA role is redundant but
			 * Safari+VoiceOver won't announce the list otherwise.
			 */
			/* eslint-disable jsx-a11y/no-redundant-roles */ }
			<ul className="entities-saved-states__changes">
				{ globalStylesChanges.map( ( change ) => (
					<li key={ change }>{ change }</li>
				) ) }
			</ul>
			{ /* eslint-enable jsx-a11y/no-redundant-roles */ }
```

This approach currently appears to be necessary in order to read it as a list in all browsers and also avoid eslint errors.

Reply:
Thanks for highlighting this.

I did come across this once - Safari won't recognise only when the list-style is set to none, right?

See: https://developer.mozilla.org/en-US/docs/Web/CSS/list-style#accessibility_concerns

The `<ul />`s in this PR have a list-style. I'd rather avoid a11y hacks because Safari is not friendly 😄 

Having said that, I'm happy to update in a separate bugfix patch if the role hack is preferred 👍🏻